packages/auth/tsconfig.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "../../tsconfig.json"
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
```

Comment:
Add this to root `tsconfig.json`. This way all packages will get `"resolveJsonModule": true`.

Reply:
https://github.com/ttoss/modules/issues/56